pkg/porter/plugins.go

```diff
@@ -158,7 +158,7 @@ func (p *Porter) InstallPlugin(ctx context.Context, opts plugins.InstallOptions)
 
 	plugin, err := p.Plugins.GetMetadata(ctx, opts.Name)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get plugin metadata: %w", err)
```

Comment:
By the way, I created https://github.com/getporter/porter/issues/2376 to improve the error message returned when calls to the plugin fail. It would have helped us debug if stderr had been included in the error message that was returned in err.